source/components/drag-and-drop/elements/component-draggable/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
```

Comment:
Inclui esse componente como um "teste", para ser passado caso quiséssemos personalizar o drag in drop.